source/network/ODServer.cpp

```diff
@@ -606,6 +610,7 @@ bool ODServer::processClientNotifications(ODSocketClient* clientSocket)
             const std::string& nick = clientSocket->getPlayer()->getNick();
             int32_t id = clientSocket->getPlayer()->getId();
             int seatId = seat->getId();
+            seat->setMapSize(gameMap->getMapSizeX(), gameMap->getMapSizeY());
```

Comment:
Well, this code is protected by a "if(mServerMode != ServerMode::ModeEditor) break;" 20 lines up. Is it not enough ?